### PR TITLE
ran go fmt and added .yml to a switch statement

### PR DESCRIPTION
**Pull request by sjordan8**

### CODE REVIEW

File: main.go [hunk 1]
```diff
@@ -33,6 +32,8 @@ func createManifest(ext string) (manifestInterface, error) {
 		return &manifestJSON{}, nil
 	case "yaml":
```
Comment (ChrisSchreiber) [inline main.go]: Why not just `case "yaml", "yml":`

**Review by gesparza3 — COMMENTED** [review]
The new logic to check for `.yml` file extension types looks fine to me however I'm not sure how opinionated we are with Go formatting. Our other projects(_ex: Rode_) use two spaces for indentation so hopefully we can keep it consistent going forward. Would like some input from @ChrisSchreiber and @mrparkers

File: main.go (hunk 1 above)
Comment (sjordan8) [reply main.go]: I did not know that was a thing, I'll change it.

**Comment by ChrisSchreiber** [comment]
@gesparza3 The recommendation for go, which `go fmt` uses, is tabs instead of spaces :face_vomiting:

**Comment by sjordan8** [comment]
@gesparza3 @ChrisSchreiber I would prefer to use `go fmt`'s style because I like to use it as a linter. But if everyone else is against me I'm totally fine with keeping everything as two spaces

**Review by ChrisSchreiber — APPROVED** [review]

**Comment by mrparkers** [comment]
All of our codebases that use go should adhere to the `go fmt` style imo.

**Pull request merged**